abr_jaco2/interface/jaco2_rs485.cpp

```diff
@@ -334,40 +363,46 @@ void Jaco2::InitPositionMode() {
 void Jaco2::InitForceMode() {
     // STEP 1: Get current position
     log_msg(1, "Initializing force mode");
-    log_msg(1, "STEP 1/7: Getting current position...");
+    log_msg(1, "STEP 1/8: Getting current position...");
     SendAndReceive(get_position_message, true);
 
     // STEP 2-4: set control parameters
     // Let's also try setting the static friction parameter
-    log_msg(1,"STEP 2/7: Setting torque config feedforward advanced parameters...");
+    log_msg(1,"STEP 2/8: Setting torque config feedforward advanced parameters...");
     // no need for a response, because I have no idea what's supposed to be
     // returned, this is lacking a lot of documentation
     SendAndReceive(torques_config_feedforward_advanced_message, false);
 
     // Set advanced torque parameters 1
-    log_msg(1, "STEP 3/7: Setting advanced torque parameters 1...");
+    log_msg(1, "STEP 3/8: Setting advanced torque parameters 1...");
     // no need for a response, because I have no idea what's supposed to be
     // returned, this is lacking a lot of documentation
     SendAndReceive(torque_config_parameters_message1, false);
 
     // Set advanced torque parameters 2
-    log_msg(1, "STEP 4/7: Setting advanced torque parameters 2...");
+    log_msg(1, "STEP 4/8: Setting advanced torque parameters 2...");
     // no need for a response, because I have no idea what's supposed to be
     // returned, this is lacking a lot of documentation
     SendAndReceive(torque_config_parameters_message2, false);
 
+    // Set torque control limits
+    log_msg(1, "STEP 5/8: Setting torque control limits...");
+    // no need for a response, because I have no idea what's supposed to be
+    // returned, this is lacking a lot of documentation
+    SendAndReceive(torque_control_limits_message, false);
+
     // STEP 5: Set torque safety parameters
-    log_msg(1, "STEP 5/7: Setting torque safety parameters...");
+    log_msg(1, "STEP 6/8: Setting torque safety parameters...");
     SendAndReceive(safety_message, true);
 
     int joints_updated0 = 0;
     while (joints_updated0 < 6) {
         // STEP 6: Send torque values to compare with sensor readings
-        log_msg(1, "STEP 6/7: Checking torque sensor calibration...");
+        log_msg(1, "STEP 7/8: Checking torque sensor calibration...");
         int joints_updated1 = 0;
         while(joints_updated1 < 6) {
             // NOTE: motor 1 is flipped so input torques need to be * - 1
-            torque_load[1] *= -1;
+            // torque_load[1] *= -1;
```

Comment:
why was this taken out? did something change elsewhere?

Reply:
I think it's because the output torque signal is already flipped and this is left over from testing. I missed it as I was going through. 
![Figure_1 (1)](https://user-images.githubusercontent.com/14222591/57701097-3f585100-7629-11e9-969f-37ba960df4d6.png)